test/path_test.cpp

```diff
@@ -121,33 +123,33 @@ void CheckPath(Point startPosition, Point destinationPosition, std::vector<int8_
 	// EXPECT_EQ(startPosition, destinationPosition) << "Path doesn't lead to destination";
 }
 
-TEST(PathTest, FindPath)
-{
-	CheckPath({ 8, 8 }, { 8, 8 }, {});
-
-	// Traveling in cardinal directions is the only way to get a first step in a cardinal direction
-	CheckPath({ 8, 8 }, { 8, 6 }, { 1, 1 });
-	CheckPath({ 8, 8 }, { 6, 8 }, { 2, 2 });
-	CheckPath({ 8, 8 }, { 10, 8 }, { 3, 3 });
-	CheckPath({ 8, 8 }, { 8, 10 }, { 4, 4 });
-
-	// Otherwise pathing biases along diagonals and the diagonal steps will always be first
-	CheckPath({ 8, 8 }, { 5, 6 }, { 5, 5, 2 });
-	CheckPath({ 8, 8 }, { 4, 4 }, { 5, 5, 5, 5 });
-	CheckPath({ 8, 8 }, { 12, 20 }, { 7, 7, 7, 7, 4, 4, 4, 4, 4, 4, 4, 4 });
-}
+// TEST(PathTest, FindPath)
+// {
+// 	CheckPath({ 8, 8 }, { 8, 8 }, {});
+
+// 	// Traveling in cardinal directions is the only way to get a first step in a cardinal direction
+// 	CheckPath({ 8, 8 }, { 8, 6 }, { 1, 1 });
+// 	CheckPath({ 8, 8 }, { 6, 8 }, { 2, 2 });
+// 	CheckPath({ 8, 8 }, { 10, 8 }, { 3, 3 });
+// 	CheckPath({ 8, 8 }, { 8, 10 }, { 4, 4 });
+
+// 	// Otherwise pathing biases along diagonals and the diagonal steps will always be first
+// 	CheckPath({ 8, 8 }, { 5, 6 }, { 5, 5, 2 });
+// 	CheckPath({ 8, 8 }, { 4, 4 }, { 5, 5, 5, 5 });
+// 	CheckPath({ 8, 8 }, { 12, 20 }, { 7, 7, 7, 7, 4, 4, 4, 4, 4, 4, 4, 4 });
+// }

```

Comment:
The intent for these tests is to ensure the pathfinding algo is consistent/deterministic. If you get this implementation performant enough that you're happy with it and it gives a consistent path between two points on a fixed grid I'm happy to have the list of nodes here changed to match whatever the output ends up giving.

Reply:
Oh yeah I didn't mean to comment this out